src/data_writing.py

```diff
@@ -0,0 +1,29 @@
+
+"""
+This module is used to write out the results.
+"""
+
+import csv
+import os
+
+class Write:
+    def __init__(self):
+        pass
+
+    def write(self, data_dict):
```

Comment:
docstrings

Reply:
docstrings should also contain information about the parameters and the return value